### ISSUE TITLE
UltraGCN: Ultra Simplification of Graph Convolutional Networks for Recommendation

### ISSUE THREAD

**Issue by HuilinChenJN** [HuilinChenJN/study_record#8]
# Title: UltraGCN: Ultra Simplification of Graph Convolutional Networks for Recommendation
## Motivation:  
The core of GCNs lies in its message passing mechanism to aggregate neighborhood information. However, we observed that message passing largely slows down the convergence of GCNs during training, especially for large-scale recommender systems.
Therefore, the paper designs an ultra-simplified formulation of GCNs, which skips infinite layers of message passing for efficient recommendations.
## Paper Goal
The aim of this paper is directly approximate the limit of infinite-layer graph convolutions via a constraint loss. And allows for more appropriate edge weight assignments and flexible adjustment of the relative importance among different types of relationships.

## Previous Work Limits
Those methods tend to seek for more sophisticated neural network encoders. However, the burdensome message passing operations dominate their training, which leads to a time-consuming training process for CF on the large-scale graph.
